server/netty-server/src/main/scala/sttp/tapir/server/netty/internal/StreamCompatible.scala

```diff
@@ -0,0 +1,22 @@
+package sttp.tapir.server.netty.internal
+
+import sttp.capabilities.Streams
+import io.netty.handler.codec.http.HttpContent
+import org.reactivestreams.Publisher
+import java.io.InputStream
+import sttp.tapir.TapirFile
+import sttp.tapir.FileRange
+
+private[netty] trait StreamCompatible[S <: Streams[S]] {
```

Comment:
shouldn't we have an impl for cats as well?

Reply:
Yes, as I mentioned in the PR description, I'd like to make refactoring for CE in another PR, so that the scope of this PR is ZIO-only.

Reply:
ah, of course, I didn't read the description ;)